### PR TITLE
Feast diagrams

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 1]
```diff
@@ -9,6 +9,8 @@ You will setup an offline feature store with PostgreSQL and an online feature st
 Then, populate the offline feature store using Jet jobs.
 Finally, you will transfer the features in the offline store to the online store.
 
+image:ROOT:feast_batch.png[Feast batch wokflow]
```
Comment (oliverhowell) [inline docs/modules/integrate/pages/feature-engineering-with-feast.adoc:12]:
```suggestion
.Overview of Feast batch workflow 
image:ROOT:feast_batch.png[Feast batch workflow]
```

**Review by oliverhowell — CHANGES_REQUESTED** [review]
Added backport to 5.5 label (correct?) and a couple of typos on label.
Have also added image labels to see what it looks like. If we want to remove 'Figure 1:' part then we need to add `:figure-caption!: `to the start of the page
as per https://docs.asciidoctor.org/asciidoc/latest/macros/images/

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc [hunk 2]
```diff
@@ -8,6 +8,8 @@
 You will setup an offline feature store with PostgreSQL and an online feature store with Hazelcast.
 Then update the online feature store using a Jet job in real time from transactions streaming in a Kafka topic.
 
+image:ROOT:feast_streaming.png[Feast streaming wokflow]
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/integrate/pages/streaming-features-with-feast.adoc:12]: Updated at:
https://github.com/hazelcast/hz-docs/pull/1288/commits/bc1d385cf39c084d7f7cd6c27c3e6213b7731136

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/integrate/pages/feature-engineering-with-feast.adoc:13]: Updated at:
https://github.com/hazelcast/hz-docs/pull/1288/commits/bc1d385cf39c084d7f7cd6c27c3e6213b7731136

**Review by oliverhowell — APPROVED** [review]
Thanks @yuce

**Pull request merged**